src/main/kotlin/no/nav/syfo/BootstrapApplication.kt

```diff
@@ -64,8 +66,11 @@ fun main() {
                 val dkifConsumer = getDkifConsumer(env.urlEnv, azureAdTokenConsumer, stsConsumer)
                 val oppfolgingstilfelleConsumer = getSyfosyketilfelleConsumer(env.urlEnv, stsConsumer)
                 val sykmeldingerConsumer = SykmeldingerConsumer(env.urlEnv, azureAdTokenConsumer)
+                val narmesteLederConsumer = NarmesteLederConsumer(env.urlEnv, azureAdTokenConsumer)
+                val arbeidsgiverNotifikasjonProdusentConsumer = ArbeidsgiverNotifikasjonProdusentConsumer(env.urlEnv, azureAdTokenConsumer, narmesteLederConsumer)
 
                 val beskjedKafkaProducer = BeskjedKafkaProducer(env)
+                val dineSykmeldteHendelseKafkaProducer = DineSykmeldteHendelseKafkaProducer(env)
```

Comment:
Kan du sende denne til `varselBusModule` også, så slipper vi å opprette den to ganger

Reply:
Enig